extensions/deepcast/tsconfig.json

```diff
@@ -1,16 +1,19 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
-  "display": "Node 16",
-  "include": ["src/**/*"],
+  "display": "Node 18",
+  "include": ["src/**/*", "raycast-env.d.ts"],
   "compilerOptions": {
-    "lib": ["es2020"],
+    "lib": ["es2021"],
     "module": "commonjs",
-    "target": "es2020",
+    "target": "es2021",
     "strict": true,
     "isolatedModules": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "resolveJsonModule": true,
+    "typeRoots": ["."],
+    "types": ["raycast-env"]
```

Comment:
This was added to take advantage of the autogenerated types in `raycast-env.d.ts`. `ray dev` uses them just fine, but the editor needs to know that it exists by adding these lines.

Reply:
I'm not sure I understand why you need this.

If something is not found, you should be able to run `> ESLint: Restart ESLint Server` in code.